basic_test.go

```diff
@@ -455,11 +455,6 @@ func TestCloseBlocking(t *testing.T) {
 	sub.Close() // cancel sub
 }
 
-func panicOnTimeout(d time.Duration) {
-	<-time.After(d)
-	panic("timeout reached")
-}
```

Comment:
This would _always_ panic, no matter if there was a timeout or not. The only reason this didn't show up before is that the test suite would finish before we reached the 5s timeout.

With shuffling of tests (`go test -shuffle`), this is not a valid assumption any more. Besides, `panic`king is a very bad way to end a failing test anyway.